src/logDownloaderService.py

```diff
@@ -0,0 +1,79 @@
+import os
+import time
+import hashlib
+import subprocess
+import psutil
+from git import Repo
+import shutil
+from loguru import logger
+
+LOGS_DIRECTORY = os.getenv('LOGS_DIRECTORY', '/path')
```

Comment:
I would crash instead of using a default directory which probably doesn't exist.